app/ai/voice/agents/breeze_buddy/mcp/client.py

```
@@ -0,0 +1,164 @@
+            args: Arguments for the command
+            env: Environment variables to set
+            timeout: Optional timeout override
+
+        Returns:
+            True if connection successful, False otherwise
+        """
+        if self._connected:
+            logger.warning("MCP client already connected, disconnecting first")
+            await self.disconnect()
+
+        timeout_seconds = timeout or DEFAULT_MCP_TIMEOUT
```

Ruff (0.15.2)

[error] Local variable `timeout_seconds` is assigned to but never used

Remove assignment to unused variable `timeout_seconds`

(F841)